docs/xdr/usecases/playbook/Get_events_information_from_alert.md

```diff
@@ -0,0 +1,39 @@
+# Get events information from alert
+
+This use case describes how to get the Mac Address of an alert. 
+
+## Prerequisites
+
+- A SEKOIA.IO XDR licence
+- The user that configures the Playbook should have a role that contains:
+	* The SYMPHONY permissions
+- An API Key with a role that contains at least the following permissions:
+	* `SIC_READ_ALERTS`
+	* `EVENTS_READ_STATS`
+	* `EVENTS_READ_DATASETS`
+	* `SIC_READ_EVENT_STATS`
+
+!!! note 
+    To create your API Key, follow this [documentation](../../../getting_started/generate_api_keys.md).
+    
+## Playbook configuration
+
+Here is the configuration below: 
+
+![Playbook Enrich_information_in_alert](docs/assets/playbooks/library/UseCases/Enrich_information_in_alert.png)
```

Comment:
image is available [here](https://github.com/SEKOIA-IO/documentation/tree/main/docs/assets/playbooks/library/UseCases)
As this page is not yet linked in the documentation tree, it is not displayed

It will be displayed when this PR will be validated and that I link this page to the documentation